scipy/stats/stats.py

```diff
@@ -2880,6 +2893,10 @@ def median_absolute_deviation(x, axis=0, center=np.median, scale=1.4826,
         med = np.apply_over_axes(center, arr, axis)
         mad = np.median(np.abs(arr - med), axis=axis)
 
+    # NOTE: _scale_conversions was for IQR so for MAD we need to divide by 2
+    if scale_key == 'normal':
```

Comment:
Also I'm adding some tests and need to rethink this part, the `scale_key` isn't assigned/instantiated yet here if `scale=1.0`

Reply:
I switched this around a bit, check only once is scale is string and handle appropriately.